wpilibj/src/main/java/edu/wpi/first/wpilibj/DataLogManager.java

```diff
@@ -213,8 +213,8 @@ private static String makeLogDir(String dir) {
       try {
         // prefer a mounted USB drive if one is accessible
         Path usbDir = Paths.get("/u").toRealPath();
-        if (Files.isWritable(usbDir)) {
-          return usbDir.toString();
+        if (Files.isWritable(usbDir) && new File("/u/logs").mkdir()) {
```

Comment:
[`File.mkdir()`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/File.html#mkdir()) returns false if the directory already exists.

Reply:
Good catch, thanks.